File: w09/d03/matt_lucas/mail_objects_app/spec/javascripts/mailitem_spec.js
describe("a mailItem", function() {

  it("is created with a shipToAddress", function() {
    mailitem = new mailItem('new york');
    expect(mailitem.shipToAddress).toBe('new york');
  });
  it("is created with a stampValue", function() {
    mailitem = new mailItem('new york', '50');
    expect(mailitem.stampValue).toBe('50');
  });

});

File: w09/d01/katherine_santiago/mail_item/spec/mailitem_spec.rb
require_relative 'spec_helper'
require_relative '../lib/mailitem'

describe Mailitem do

  it "should have a ship_to address" do
    mail1 = Mailitem.new(ship_to_address: "New York")
    expect(mail1.ship_to_address).to eq("New York")
  end

  it "should have a stamp_value" do
    mail = Mailitem.new(ship_to_addess: "New York", stamp_value: 1)
    expect(mail.stamp_value).to eq(1)
  end
end